src/views/SignIn/SignIn.js

```diff
@@ -81,55 +87,65 @@ const useStyles = makeStyles(theme => ({
     paddingLeft: theme.spacing(2),
     paddingRight: theme.spacing(2)
   },
-  logoImage: {
-    marginLeft: theme.spacing(4)
-  },
   contentBody: {
     flexGrow: 1,
     display: 'flex',
-    alignItems: 'center',
+    alignItems: 'flex-start',
     [theme.breakpoints.down('md')]: {
       justifyContent: 'center'
     }
   },
   form: {
+    paddingTop: 120,
     paddingLeft: 100,
     paddingRight: 100,
-    paddingBottom: 125,
     flexBasis: 700,
     [theme.breakpoints.down('sm')]: {
       paddingLeft: theme.spacing(2),
       paddingRight: theme.spacing(2)
     }
   },
+  contentFooter: {
+    display: 'flex',
+    alignItems: 'flex-end'
+  },
   title: {
     marginTop: theme.spacing(3)
   },
   socialButtons: {
     marginTop: theme.spacing(3)
   },
-  socialIcon: {
-    marginRight: theme.spacing(1)
+  contact: {
+    marginLeft: '10px',
+    marginTop: '5px'
   },
-  loginButton: {
-    marginTop: theme.spacing(1)
+  formControl: {
+    fontSize: '11px'
   },
-  sugestion: {
-    marginTop: theme.spacing(2)
-  },
-  textField: {
-    marginTop: theme.spacing(2)
-  },
-  signInButton: {
-    margin: theme.spacing(2, 0)
+  languageSelect: {
+    fontSize: '11px'
   }
 }))
 
-const SignIn = (props) => {
-  const { t } = useTranslation()
+const SignIn = () => {
+  const { t, i18n } = useTranslation()
 
   const classes = useStyles()
 
+  const changeLanguage = (language) => {
```

Comment:
It can be interesting to extract this method inside a utility file. I think about the UserInfo component which certainly could reuse these lines.